src/main/features/dashboard/helpers/claimStatusFlow.ts

```diff
@@ -11,13 +38,94 @@ export class ClaimStatusFlow {
     next: [
       {
         description: 'Claim Issued',
-        isValidFor: (claim) => !claim.response,
+        isValidFor: (claim) => !claim.response && !claim.moreTimeRequested,
         dashboard: 'claim_issued',
         next: []
+      },
+      {
+        description: 'More Time Requested',
+        isValidFor: (claim) => !claim.response && claim.moreTimeRequested,
+        dashboard: 'more_time_requested',
+        next: []
+      },
+      {
+        description: 'Claim is Full Admission',
+        isValidFor: (claim) => claim.isFullAdmission,
+        next: [
+          {
+            ...payImmediately,
+            dashboard: 'pay_immediately',
+            next: [{
+              ...paymentPastDeadline,
+              dashboard: 'payment_past_deadline',
+              next: []
+            }]
+          },
+          {
+            ...payBySetDate,
+            dashboard: 'pay_by_set_date',
+            next: []
```

Comment:
Can't a pay-by-set-date response also be past its deadline? Maybe an instalments one too?

Reply:
It should be, but I couldn't easily trace it in the code (or in the coggle) - so I decided to slowly inch forward. I think most cases will come up as we keep the refactoring going :)